src/irisnet/noncore/cutestuff/httppoll.cpp

```diff
@@ -487,9 +487,9 @@ void HttpProxyPost::post(const QString &proxyHost, int proxyPort, const QUrl &ur
 #endif
     if (d->sock.state() != QAbstractSocket::ConnectingState) { // in case of http/1.1 it may be connected
         if (d->lastAddress.isNull()) {
-            d->sock.connectToHost(proxyHost, proxyPort);
+            d->sock.connectToHost(proxyHost, quintptr(proxyPort));
```

Comment:
has to be quint16.
or maybe we need to have `HttpProxyPost::post(const QString &proxyHost, quint16 proxyPort`